vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -839,12 +840,17 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
                     postError: (error: Error, type?: MessageErrorType): void => {
                         this.postError(error, type)
```

Comment:
```suggestion
                        this.cancelSubmitOrEditOperation()
                        this.postError(error, type)
```
So that it won't stuck in loading?